File: java/Hello.java
import java.util.Date;
import java.util.Random;
import java.sql.Timestamp;

public class Hello {
	public static void printHtml() {
		System.out.print("Content-type: text/html\n\n");
		System.out.println("<!DOCTYPE html>");
		System.out.println("<html lang=\"en\">");
		printHead();
		printBody();
		System.out.println("</html>");
	}

	public static void printHead() {
		System.out.println("  <head>");
		System.out.println("    <meta charset=\"UTF-8\">");
		System.out.println("    <title>Hello Java Example</title>");
		System.out.println("  </head>");
	}

	public static void printBody() {
		System.out.println("  <body style=\"background-color:" + getRandomColor() + "\">");
		System.out.println("    <p>Hello World from Java @ " + getTimestamp() + "</p>");
		System.out.println("  </body>");
	}

	public static String getRandomColor() {
		String[] colors = {"aqua", "black", "blue", "fuchsia", "gray",
							"green", "lime", "maroon", "navy", "olive",
							"purple", "red", "silver", "teal", "white",
							"yellow"};

		Random r = new Random();
		int num = r.nextInt(colors.length);
		return colors[num]; 
	}

	public static String getTimestamp() {
		Date date = new Date();
		return date.toString();
	}

	public static void main(String[] args) {
		printHtml();
	}
}

File: php/hello.php
<?php

	$listOfColors = array("aqua", "black", "blue", "fuchsia", "gray", "green", "lime", "maroon", "navy", "olive", "purple", "red", "silver", "teal", "white", "yellow");
	$randomNumber = rand(0,15);
	printHelloWorldRandomColor($listOfColors[$randomNumber]);

	function printHelloWorldRandomColor($randomColor)
	{
		echo "<!DOCTYPE html>
		  	<html lang='en'>
			  	<head>
			  		<meta charset='utf-8'>
	    	  		<title>Hello PHP Example</title>
	    	    </head>
	    	    <body style ='background-color:". $randomColor."'".">";
		echo "<p>Hello World from PHP @ ". time() ."</p>
		        </body>
		    </html>";
	}

?>
